usr/share/rear/build/default/502_include_mdadm_conf.sh

```diff
@@ -0,0 +1,9 @@
+grep -q blocks /proc/mdstat 2>/dev/null || return 0
+
+# Include /etc/mdadm.conf without building arrays automatically
```

Comment:
@rmetrich 
please add a comment here with the URL to your
great explanation about the reason behind like
<pre>
# for the reason behind see
# https://github.com/rear/rear/issues/1722#issuecomment-394746478
</pre>
so that later also others who may not know the details about
how multipath things behave (like me in a few month ;-)
get a starting point to understand why that is done,
cf. "Code should be easy to understand" at
https://github.com/rear/rear/wiki/Coding-Style

Many thanks in advance and now I think I understand
this pull request sufficiently so that I can fully approve it!